rosetta/text/vw_helpers.py

```diff
@@ -147,8 +155,8 @@ def find_start_line_lda_predictions(predictions_file, num_topics):
     return start_line
 
 
-def parse_lda_predictions(
-    predictions_file, num_topics, start_line, normalize=True):
+def parse_lda_predictions(predictions_file, num_topics, start_line,
```

Comment:
It might make sense to give `num_topics` and `start_line` default values here.  E.g. 10 and 0.  As long as you don't change the arg order (i.e. only add defaults, don't change order), it won't break anything.